src/interfaces/filing-interfaces/alterationFiling-interface.ts

```diff
@@ -1,15 +1,16 @@
-import { AlterationIF } from '@/interfaces'
+import { AlterationIF, BusinessSnapshotIF } from '@/interfaces'
```

Comment:
`BusinessSnapshotIF` is for future use? (Is not used here atm.)

Reply:
oh thanks good catch! 

Initially i was planning to store the original snapshot within the DRAFT alteration filings, but after our discussion we determined we didn't need too and could GET the current information.

Reply:
Removed in next commit!